src/client/ui/highlight.js

```diff
@@ -0,0 +1,26 @@
+export default function highlight(data) {
```

Comment:
Okay. Then that's fine.
Try to make some comments on why we are using it and so on.